app/src/main/java/com/readrops/app/activities/AddAccountActivity.java

```diff
@@ -68,6 +68,9 @@ protected void onCreate(Bundle savedInstanceState) {
                 binding.providerImage.setImageResource(accountType.getIconRes());
                 binding.providerName.setText(accountType.getName());
                 binding.addAccountName.setText(accountType.getName());
+                if (accountType.getName() == R.string.freshrss) {
```

Comment:
Please compare only account type with `accountType == AccountType.FRESHRSS` instead of the account type name.

Reply:
Done